network-metrics-nuttcp.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
```

Comment:
In fact, I can't run it from root, but can from under `tests/metrics/network` - due to the `source "${SCRIPT_PATH}/../../lib/test-common.bash"`. @GabyCT I guess you tested this in a different tree before a copy/merge/push cycle? :-(